### PR TITLE
Filter WARN message from Netty when it can't parse the /etc/hosts file

### LINKED ISSUE #5647: Error message on start (/etc/hosts permissions)
**Describe the bug**
At start of stock quarkus without any extensions I get the following message:

2019-11-21 07:25:46,065 WARN  [io.net.res.HostsFileParser] (main) Failed to load and parse hosts file at /etc/hosts: java.io.FileNotFoundException: /etc/hosts (Permission denied)

The application is still working

**Expected behavior**

No superfluous error message.

**Actual behavior**

WARN  [io.net.res.HostsFileParser] (main) Failed to load and parse hosts file at /etc/hosts: java.io.FileNotFoundException: /etc/hosts (Permission denied)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at io.netty.resolver.HostsFileParser.parse(HostsFileParser.java:132)
	at io.netty.resolver.HostsFileParser.parseSilently(HostsFileParser.java:89)
	at io.netty.resolver.HostsFileParser.parseSilently(HostsFileParser.java:76)
	at io.vertx.core.impl.resolver.DnsResolverProvider.<init>(DnsResolverProvider.java:123)
	at io.vertx.core.spi.resolver.ResolverProvider.factory(ResolverProvider.java:39)
	at io.vertx.core.impl.AddressResolver.<init>(AddressResolver.java:75)
	at io.vertx.core.impl.VertxImpl.<init>(VertxImpl.java:167)
	at io.vertx.core.impl.VertxImpl.vertx(VertxImpl.java:92)
	at io.vertx.core.impl.VertxFactoryImpl.vertx(VertxFactoryImpl.java:40)
	at io.vertx.core.impl.VertxFactoryImpl.vertx(VertxFactoryImpl.java:32)
	at io.vertx.core.Vertx.vertx(Vertx.java:85)

**To Reproduce**
1. mvn io.quarkus:quarkus-maven-plugin:1.0.0.CR2:create  (the same behaviour on CR1)
2.  mvn compile quarkus:dev

**Configuration**
```properties
# Add your application.properties here, if applicable.

```

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Darwin 19.0.0 Darwin Kernel Version 19.0.0, Mac OS Catalina
 - Output of `java -version`: 
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)

 - GraalVM version (if different from Java): 
 - Quarkus version or git rev: CR2

**Additional context**

`sudo chmod a+r /etc/hosts`fixes the problem

### PR DESCRIPTION
**Pull request by jaikiran**
Fixes https://github.com/quarkusio/quarkus/issues/5647

Netty logs a `WARN` message (along with the exception stacktrace) when it can't parse the hosts file. The commit here filters out that message using the `LogCleanupFilterBuildItem`.

No automated tests added, but manual testing showed that this change helped prevent that `WARN` message showing up in the logs.

### CODE REVIEW

**Review by geoand — APPROVED** [review]

**Comment by gsmet** [comment]
Let's wait for @cescoffier before merging because I wonder what could be the consequences of having the hosts entirely ignored. That means that you don't have a correspondance for localhost typically.

Totally ignoring this warning might lead to some issues.

**Comment by geoand** [comment]
I agree with waiting for the expert which is why I didn't merge, just approved.

**Comment by cescoffier** [comment]
It should be fine.

**Review by cescoffier — APPROVED** [review]

**Pull request merged**
